corehq/apps/app_manager/urls.py

```diff
@@ -128,6 +130,7 @@
     url(r'^download/(?P<app_id>[\w-]+)/',
         include('corehq.apps.app_manager.download_urls')),
     url(r'^formdefs/(?P<app_id>[\w-]+)/', 'formdefs', name='formdefs'),
+    url(r'^ng_template/(?P<template>[\w-]+)', DynamicTemplateView.as_view(), name='ng_template'),
```

Comment:
Ah, ok...that's what you were trying to do (re: my comment above). That seems ok then. Although I'm not really sure how much I like the idea of both the sidebar and the main content loading together...does the sidebar actually have to change much, if at all? Could there be a separate template for sidebar content and another for the main content?